lib/commands/get.js

```diff
@@ -1,46 +1,67 @@
+/* global module, require */
 'use strict';
 
+let ReadConfigTask = require('../tasks/read-config');
 let filesystem = require('../utils/filesystem');
+let Promise = require('ember-cli/lib/ext/promise');
```

Comment:
I know was me that did the error first. But doing the salesforce integration I learned that what get from the require should be const and not let. would you mind to change it please? thanks 🙏